consensus/bor/heimdallgrpc/milestone.go

```diff
@@ -24,9 +24,10 @@ func (h *HeimdallGRPCClient) FetchMilestoneCount(ctx context.Context) (int64, er
 	return res.Result.Count, nil
 }
 
-func (h *HeimdallGRPCClient) FetchMilestone(ctx context.Context) (*milestone.Milestone, error) {
+func (h *HeimdallGRPCClient) FetchMilestone(ctx context.Context, number int64) (*milestone.Milestone, error) {
 	h.logger.Info("Fetching milestone")
 
+	// TODO: use number
```

Comment:
do we need to sort this one out before merging into devel?
depends on whether http or graph is used when running heimdall and erigon on the same process

Reply:
in `etc/backend.go` we have this which means we can't merge in devel until the grpc is changed?

```
        var heimdallClient heimdall.IHeimdallClient
	if chainConfig.Bor != nil {
		if !config.WithoutHeimdall {
			if config.HeimdallgRPCAddress != "" {
				heimdallClient = heimdallgrpc.NewHeimdallGRPCClient(config.HeimdallgRPCAddress, logger)
			} else {
				heimdallClient = heimdall.NewHeimdallClient(config.HeimdallURL, logger)
			}
		}

		flags.Milestone = config.WithHeimdallMilestones
	}
```

Reply:
Actually just noticed the change is backward compatible so only the new Astrid package won't work with the grpc which is not a blocker to merge this. We would need to update those protos and implementations at some point soonish though.

Reply:
yes, I've added to my TODO list for later